src/RecoveryServices/RecoveryServices.SiteRecovery/Models/PSObjects.cs

```diff
@@ -2370,6 +2388,24 @@ public class ASRVMNicConfig
         [DataMember]
         public string TfoVMNetworkId { get; set; }
 
```

Comment:
You've added the properties here but how will they be populated?
We have provided the New-ASRVmNicConfig cmdlet to help populate these. You'll need to update that and update the markdown for the same.

Reply:
handled